src/main/java/frc/robot/commands/MotorTestCommand.java

```diff
@@ -0,0 +1,80 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+import frc.robot.subsystems.RollerSubsystem;
+import frc.robot.subsystems.TiltedElevatorSubsystem;
+import frc.robot.subsystems.TiltedElevatorSubsystem.ElevatorState;
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
+
+public class MotorTestCommand extends InstantCommand {
```

Comment:
maybe this shouldn't be an InstantCommand? i think currently this will lock up the CommandScheduler bc there are delays/sleeps in the command itself.

I think you could fix this by either using sequentialcommands or writing your own command that checks the time since starting and decides what to do based on that every time its update()ed.

https://docs.wpilib.org/en/stable/docs/software/commandbased/commands.html